lua/sql.lua

```diff
@@ -480,23 +385,24 @@ end
 ---@usage db:get("todos", { where = { id = 1 })
 ---@usage db:get{ todos = { where = { id = 1 } }}
 -- @return lua list of matching rows
-function sql:get(...)
+function sql:select(...)
+  local method = "select"
   local args = {...}
   local ret_vals = {}
 
   local inner_eval = function(tbl, p)
-    return self:eval(self:parse("select", {
-      tbl = tbl,
+    local sqlstmt = parse(tbl, method, {
       select = p and p.select or nil,
       where = p and p.where or nil,
       join = p and p.join or nil,
-      placeholders = true,
-    }), p and p.where)
+      named = true,
+    })
+    return self:eval(sqlstmt)
   end
 
   if u.is_str(args[1]) then
     local tbl = args[1]
-    assert_tbl(self, tbl)
+    assert_tbl(self, tbl, "select/get/find")
```

Comment:
Just use select. Also do we need an `assert_tbl` down in the elseif part?

Reply:
👍